.gitignore

```diff
@@ -1,3 +1,19 @@
-/build-runv
+*.patch
+*.o
+*.swp
 /kata-runtime
 /data/kata-collect-data.sh
+/cli/coverage.html
+/cli/config-generated.go
+/cli/data/kata-collect-data.sh
+/cli/data/kata-collect-data.sh.in
```

Comment:
These two `kata-collect-data.sh` lines aren't needed as the script now lives in the top level `./data/` directory (see one of the lines above for the new location).